supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchJobPaths.scala

```diff
@@ -55,19 +53,4 @@ final case class AwsBatchJobPaths(override val workflowPaths: AwsBatchWorkflowPa
   // at the time that code runs.
   override def defaultStdoutFilename: String = s"$logBasename-stdout.log"
   override def defaultStderrFilename: String = s"$logBasename-stderr.log"
-
```

Comment:
This was all copypasta from the PAPI backends. that `foo.log` file is the PAPI log. What'd happen is that this would be inserted into the custom metadata but not exist when files would get copied over